File: Lista3 - py/Exercicios-Casa/Exercicio10.py
while True:
    OP = input("Digite alguma operação aritmética('+','-','*','/' ou 'S' - Sair): ")
    if OP == "S":
        break
    A = float(input("Digite o primeiro numero: "))
    B = float(input("Digite o segundo numero: "))
    if OP == "+":
        print(A+B)       
    elif OP == "-":
        print(A-B)       
    elif OP == "*":
        print(A*B)
    elif OP == "/":
        print(A/B)

File: Lista3 - C/ExerciciosCasa/Exercicio10.c
#include <stdio.h>
#include <stdbool.h>
int main()
{   
    char OP;
    float A,B;
    while (true){
        printf("Digite alguma operação aritmética('+','-','*','/' ou 'S' - Sair): ");
        scanf(" %c", &OP);
        if(OP == 'S'){
            break;
        }
        printf("Digite o primeiro numero: ");
        scanf("%f", &A);
        printf("Digite o segundo numero: ");
        scanf("%f", &B);
        switch(OP){
            case '+':
                printf("%.2f\n",A+B);
                break;
            case '-':
                printf("%.2f\n",A-B);
                break;
            case '*':
                printf("%.2f\n",A*B);
                break;
            case '/':
                printf("%.2f\n",A/B);
                break;
        }
    }
}